library/std/src/sys/solid/env.rs

```diff
@@ -2,8 +2,8 @@ pub mod os {
     pub const FAMILY: &str = "itron";
     pub const OS: &str = "solid";
     pub const DLL_PREFIX: &str = "";
```

Comment:
Does this need to be "lib"?

Reply:
No, this is correct. This platform's convention for library names is: `XXX.so` and `libXXX.a`.